jep/201/README.adoc

```diff
@@ -232,3 +232,4 @@ https://github.com/jenkinsci/configuration-as-code-plugin
 == References
 
 This topic was initially discussed on https://issues.jenkins-ci.org/browse/JENKINS-31094[JENKINS-31094].
+Subsequent discussion [here](https://groups.google.com/d/msgid/jenkinsci-dev/CANfRfr2BpTRM40fMWHG_7RhQfWdNocomzmkR7%3DdR0xO8SqkUYw%40mail.gmail.com).
```

Comment:
This is AsciiDoc, not Markdown.

Also, the canonical URL to the thread is https://groups.google.com/d/msg/jenkinsci-dev/6TjlxEqHUEs/nKSG1xSkCQAJ

Reply:
Fixed